openupgradelib/openupgrade.py

```diff
@@ -267,11 +274,45 @@ def load_data(cr, module_name, filename, idref=None, mode='init'):
                 cr, module_name, pathname, fp.read(), idref, mode, noupdate)
         elif ext == '.yml':
             yaml_import(cr, module_name, fp, None, idref=idref, mode=mode)
+        elif mode == 'init_no_create':
+            for fp2 in _get_existing_records(cr, fp, module_name):
+                tools.convert_xml_import(
+                    cr, module_name, fp2, idref, mode=mode
```

Comment:
Change this to `init`. Please rebase also on latest branch for seeing Travis result.

Reply:
good one. done